.pre-commit-config.yaml

```diff
@@ -19,26 +19,3 @@ repos:
       - id: check-docstring-first
       - id: check-toml
       - id: check-yaml
-
-  - repo: local
-    hooks:
-      - id: pyproject.toml
-        name: validate pyproject.toml
-        language: system
-        entry: poetry check
-        files: 'pyproject\.toml'
-        pass_filenames: false
-
-      - id: black
-        name: black
-        stages: [commit]
-        language: system
-        entry: poetry run black
-        types: [python]
-
-      - id: flake8
-        name: flake8
-        stages: [commit]
-        language: system
-        entry: poetry run flake8
-        types: [python]
```

Comment:
Why this?

Reply:
the current validate gha doesn't know how to handle this

Reply:
OK. I think we should leave this in and let GHA be broken until we fix it. Removing this stuff takes away pretty much all the linting.